spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -129,8 +129,12 @@ def get_parser():
                         help='Setting for environment variable: PATH_SEGMANUAL\n'
                         'A path containing manual segmentations to be used by the script program.')
     parser.add_argument('-script-args', default='',
-                        help='A quoted string with extra flags and arguments to pass to the script. '
-                        'For example \'sct_run_batch -path-data data/ -script-args "-foo bar -baz /qux" process_data.sh \'')
+                        help='A quoted string with extra flags and arguments to pass to the script.\n'
+                             'For example \'sct_run_batch -path-data data/ -script process_data.sh '
+                             '-script-args "VARIABLE1 VARIABLE2"\'.\n'
```

Comment:
One more tiny nitpick: We might want to add a newline and some extra spaces here, to better align the arguments?

```
                             'For example \'sct_run_batch -path-data data/ -script process_data.sh\n'
                             '                           -script-args "VARIABLE1 VARIABLE2"\'.\n'
```

Before:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/9e83b2a0-59c0-42db-82d9-01cc3c66594f)

After:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/544b4a22-dc41-49d7-971c-5590e4841b16)

Reply:
Thanks for pointing this out! 
This is related to the Terminal windows size, right? If I use a full-screen terminal (which I usually do), the help is formatted fine.
I can see that this type of formatting issue is also presented for other arguments, though:
<img width="773" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/6f18d12a-87fa-472c-9778-3931d5bec32a">
Maybe the formating for all args could be addressed in a separate PR?

Reply:
Ahhh, right. Please disregard my suggestion!